### PR TITLE
Feature/add clarification basis ressources ptdata 733

### PR DESCRIPTION
**Pull request by f-peverali**
## Motivation and Context
PTDATA-733 wird hier nur teilweise gelöst.
Der Fokus der Änderung liegt auf Datenobjekt.md (https://github.com/gematik/spec-ISiK-Vitalparameter-und-Koerpermasze/pull/137/files#diff-02b61106ebff770967bcc1b664a7e0e9e4e0262a91341a918e679d674ca6974a) 

## Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)?

## Checklist:

- [ ] My code follows the code style of this IG / specification.
- [ ] My change requires a change to the documentation or narrative (intend) of the IG.
- [ ] I have already updated the documentation / narrative (intend) accordingly.

### CODE REVIEW

File: ImplementationGuide/markdown/AnmerkungenMS.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 **Bedeutung:** Der Status MUSS insbesondere zur Differenzierung von abgeschlossenen, vorläufigen oder korrigierten Beobachtungen angegeben werden. 
 Es ist möglich, dass einzelne Systeme nur Werte vom Status `final` bereitstellen. Empfangende Systeme müssen jedoch in der Lage sein, sämtliche Status-Codes zu interpretieren.
 
-**Hinweis:** Siehe Beschreibung in der [FHIR Kernspezifikation](http://hl7.org/fhir/observation-definitions.html#Observation.status).
+**Hinweis:** Siehe Beschreibung in der [FHIR Kernspezifikation]https://hl7.org/fhir/R4/observation-definitions.html#Observation.status).
```
Comment (ylboerner) [inline ImplementationGuide/markdown/AnmerkungenMS.md]:
```suggestion
**Hinweis:** Siehe Beschreibung in der [FHIR Kernspezifikation](https://hl7.org/fhir/R4/observation-definitions.html#Observation.status).
```

File: ImplementationGuide/markdown/CapabilityStatement.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
-# Konformitätserklärung [(CapabilityStatement)](http://hl7.org/fhir/capabilitystatement.html)
+# Konformitätserklärung [(CapabilityStatement)]https://hl7.org/fhir/R4/capabilitystatement.html)
```
Comment (ylboerner) [inline ImplementationGuide/markdown/CapabilityStatement.md]:
```suggestion
# Konformitätserklärung [(CapabilityStatement)](https://hl7.org/fhir/R4/capabilitystatement.html)
```

File: ImplementationGuide/markdown/CapabilityStatement.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 
-Jede Instanz eines bestätigungsrelevanten Systems MUSS an ihrem Endpunkt eine [CapabilityStatement](http://hl7.org/fhir/capabilitystatement.html)-Ressource bereitstellen.
+Jede Instanz eines bestätigungsrelevanten Systems MUSS an ihrem Endpunkt eine [CapabilityStatement]https://hl7.org/fhir/R4/capabilitystatement.html)-Ressource bereitstellen.
```
Comment (ylboerner) [inline ImplementationGuide/markdown/CapabilityStatement.md]:
```suggestion
Jede Instanz eines bestätigungsrelevanten Systems MUSS an ihrem Endpunkt eine [CapabilityStatement](https://hl7.org/fhir/R4/capabilitystatement.html)-Ressource bereitstellen.
```

File: ImplementationGuide/markdown/CapabilityStatement.md [hunk 4]
```diff
@@ -1,27 +1,27 @@
...
 
-Hierzu MUSS die [capabilities-Interaktion gemäß FHIR-Spezifikation](http://hl7.org/fhir/http.html#capabilities) unterstützt werden.
+Hierzu MUSS die [capabilities-Interaktion gemäß FHIR-Spezifikation]https://hl7.org/fhir/R4/http.html#capabilities) unterstützt werden.
```
Comment (ylboerner) [inline ImplementationGuide/markdown/CapabilityStatement.md]:
```suggestion
Hierzu MUSS die [capabilities-Interaktion gemäß FHIR-Spezifikation](https://hl7.org/fhir/R4/http.html#capabilities) unterstützt werden.
```

File: ImplementationGuide/markdown/Interaktionen.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ Folgende Suchparameter sind für das Vitalparamter-Modul relevant, auch in Kombi
     ``` GET [base]/Observation?status=http://hl7.org/fhir/observation-status|final```
     
     Anwendungshinweise: Weitere Informationen zur Suche nach "status" 
-        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](http://hl7.org/fhir/R4/search.html#token).
+        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"]https://hl7.org/fhir/R4/search.html#token).
```
Comment (ylboerner) [inline ImplementationGuide/markdown/Interaktionen.md]:
```suggestion
        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
```

File: ImplementationGuide/markdown/Interaktionen.md [hunk 6]
```diff
@@ -22,7 +22,7 @@ Folgende Suchparameter sind für das Vitalparamter-Modul relevant, auch in Kombi
     ``` GET [base]/Observation?category=http://terminology.hl7.org/CodeSystem/observation-category|vital-signs```
     
     Anwendungshinweise: Weitere Informationen zur Suche nach "status" 
-        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](http://hl7.org/fhir/R4/search.html#token).
+        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"]https://hl7.org/fhir/R4/search.html#token).
```
Comment (ylboerner) [inline ImplementationGuide/markdown/Interaktionen.md]:
```suggestion
        finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
```

File: ImplementationGuide/markdown/Interaktionen.md [hunk 7]
```diff
@@ -44,7 +44,7 @@ Folgende Suchparameter sind für das Vitalparamter-Modul relevant, auch in Kombi
     ```GET [base]/Observation?code=http://loinc.org|85354-9```
     
     Anwendungshinweise: Weitere Informationen zur Suche nach "code" 
-    finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](http://hl7.org/fhir/R4/search.html#token).
+    finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"]https://hl7.org/fhir/R4/search.html#token).
```
Comment (ylboerner) [inline ImplementationGuide/markdown/Interaktionen.md]:
```suggestion
    finden sich in der [FHIR-Basisspezifikation - Abschnitt "Token Search"](https://hl7.org/fhir/R4/search.html#token).
```

**Review by ylboerner — CHANGES_REQUESTED** [review]

**Comment by ylboerner** [comment]
@f-peverali Hier sind ein paar Markdown Links kaputt gegangen durch search and replace. Lass uns diese erst fixen vor der richtigen Review 👍

File: Resources/sushi-config.yaml [hunk 8]
```diff
@@ -5,7 +5,7 @@ applyExtensionMetadataToRoot: false
 id: Implementierungsleitfaden ISiK-Modul Vitalparameter und Körpermaße
 name: Implementierungsleitfaden ISiK-Modul Vitalparameter und Körpermaße
 status: active
-version: 3.0.1
+version: 3.0.0
```
Comment (f-peverali) [inline Resources/sushi-config.yaml]:
```suggestion
version: 3.0.1
```

**Review by patrick-werner — APPROVED** [review]

**Pull request merged**